File: paradeigma.cc
/* Παράδειγμα προγράμματος σε C++
 *
 * Για μεταγλώττιση σε Ubuntu:
 *  - ο μεταγλωττιστής είναι προεγκατεστημένος
 *  - αν δεν είναι προεγκατεστημένος, εγκατάσταση με
 *      sudo apt install build-essential
 *  - εντολή μεταγλώττισης:
 *      gcc paradeigma.cc -o paradeigma_cc
 *
 * Για εκτέλεση του προγράμματος:
 *    ./paradeigma_cc
 *
 * Αποτέλεσμα:
 *  Γεια σου, κόσμε!
 *  Εκτέλεση if: Αληθές και όχι ψευδές!
 *  Εκτέλεση βρόγχου: 0 1 2 3 4 5 6 7 8 9
 *  Αυτή είναι μια υπορουτίνα με αριθμό 5.
 */

#include <iostream>

using namespace std;

int subroutine(int number)
{
   cout << "Αυτή είναι μια υπορουτίνα με αριθμό " << number << "." << endl;

   return 0;
}

int main() {
    cout << "Γεια σου, κόσμε!" << endl;

    if (true == true)
        cout << "Εκτέλεση if: Αληθές και όχι ψευδές!" << endl;

    cout << "Εκτέλεση βρόγχου: ";
    for (int i = 0; i < 10; i++)
    {
        cout << i << " ";
    }
    cout << endl;

    subroutine(5);

    return 0;
}

File: paradeigma.php
<meta http-equiv="content-type" content="text/html; charset=UTF-8;charset=utf-8">
<?php
/* Παράδειγμα προγράμματος σε PHP
 *
 * Για εκτέλεση σε Ubuntu:
 *  - αν δεν είναι προεγκατεστημένος webserver, εγκατάσταση με
 *      sudo apt install apache2
 *
 * Για εκτέλεση του προγράμματος:
 *    Μεταφορά του αρχείου στο folder /var/www
 *    και κλήση από κάποιον browser http://localhost/paradeigma.php
 *
 * Αποτέλεσμα:
 *  Γεια σου, κόσμε!
 *  Εκτέλεση if: Αληθές και όχι ψευδές!
 *  Εκτέλεση βρόγχου: 0 1 2 3 4 5 6 7 8 9
 *  Αυτή είναι μια υπορουτίνα με αριθμό 5.
 */


function subroutine ($number)
{
   echo "Αυτή είναι μια υπορουτίνα με αριθμό $number.<br>";

}

    $i=0;

    echo "Γεια σου, κόσμε!<br>";

    if (1) {
        echo "Εκτέλεση if: Αληθές και όχι ψευδές!<br>";
	}
    echo "Εκτέλεση βρόγχου: ";
    for ($i = 0; $i < 10; $i++)
    {
        echo $i." ";
    }
    echo "<br>";

    subroutine(5);
?>
